projects/plugins/jetpack/functions.global.php

```diff
@@ -259,7 +259,7 @@ function jetpack_render_tos_blurb() {
 	printf(
 		wp_kses(
 			/* Translators: placeholders are links. */
-			__( 'By clicking the <strong>Set up Jetpack</strong> button, you agree to our <a href="%1$s" target="_blank" rel="noopener noreferrer">Terms of Service</a> and to <a href="%2$s" target="_blank" rel="noopener noreferrer">share details</a> with WordPress.com.', 'jetpack' ),
+			__( 'By clicking <strong>Set up Jetpack</strong>, you agree to our <a href="%1$s" target="_blank" rel="noopener noreferrer">Terms of Service</a> and to <a href="%2$s" target="_blank" rel="noopener noreferrer">sync your site\'s data</a> with us.', 'jetpack' ),
```

Comment:
Can we use a `’` here so we don't have to escape the `'`?

Reply:
Actually I think we should probably use that everywhere in this PR instead of `'`, I've seen Matt talk about that before

Reply:
Yes I agree, using `‘` would be the right way to go. Great catch, thanks @CodeyGuyDylan!